website/static/js/nodesPrivacy.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * Controller for changing privacy settings for a node and its children.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+var osfHelpers = require('js/osfHelpers');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var NodesPrivacyTreebeard = require('js/nodesPrivacySettingsTreebeard');
+
+//Todo (billyhunt): change url to dev or production as needed
+var API_BASE = 'http://localhost:8000/v2/nodes/';
```

Comment:
It was decided that the selector view should expand the tree when checked AND unchecked, to show that the privacy is not inherited and let users know the state of their children.  A wise woman once sang, "I believe that children are the future. . . . "